cypress/integration/loginAngular.spec.js

```diff
@@ -0,0 +1,34 @@
+import { VAR, URL } from '../const/constants';
+import { IDs } from '../page-objects/loginAngular';
+import{ HAVETEXT } from '../const/helpers';
```

Comment:
I think that this helper should be in snake_case 
```suggestion
import{ HAVE_TEXT } from '../const/helpers';
```

Reply:
Change in the branch "NewArticule"